### ISSUE TITLE
Channel and user filters no longer works after 4.0.0 IndexedDB update

### ISSUE THREAD

**Issue by themegaxandy** [Syncxv/vc-message-logger-enhanced#54]
Any filter now returns no results after the update, even after importing from json.
![image](https://github.com/user-attachments/assets/266faded-892d-4f3c-a599-d4f241f9127c)

**Comment by Syncxv** [comment]
it says your logs are empty. it looks like you didnt import your json or it failed to import. import it again and send the logs.

**Comment by themegaxandy** [comment]
Actually the image says that there are no results found, because there is a search being done in the search bar. And in the original text I said that I already imported the old json file to the new IndexedDB. I just didn't send the image without the search, because I would have to basically censor the entire image for privacy reasons, and then it would just be a red screenshot.

**Comment by Syncxv** [comment]
perhaps your using a user id as a channel id? all the filters is working for me. also try updating in the plugin settings.

![image](https://github.com/user-attachments/assets/bb36f003-0333-4b8c-ba71-e4fd08508b41)

![image](https://github.com/user-attachments/assets/4be40581-201f-451e-82b4-8f501f0a6024)

**Comment by themegaxandy** [comment]
I actually used the Open Logs for Channel button, and the Open Logs for User button, so the search bar was auto-populated.
But if it works for you, that changes things. There must be something wrong on my side. All I did was update the plugin with the update button on the plugin itself. I'll try then just rebuilding the Vencord to see if that fixes things.
Thank you for showing that it's normal for you.
